doc-content/enterprise-only/ai/ai-con.adoc

```diff
@@ -26,8 +26,7 @@ One application of pragmatic AI is in customer support. A customer files a suppo
 
 As this example shows, you can combine machine learning to extract information from data analysis and digital decisioning to model the human knowledge and experience. You can then apply mathematical optimization to schedule human assistance. This is a pattern that you can to other situations, for example a credit card dispute and credit card fraud detection.
```

Comment:
"This is a pattern that you can "apply?" to other situations. For example, a credit card dispute and credit card fraud detection."